### ISSUE TITLE
Implement GraphX API on Cassovary

### ISSUE THREAD

**Issue by lintool** [twitter/cassovary#214]
Might be fun to try and implement the GraphX API on Cassovary. Then we can compare scale up vs. scale out.

**Comment by pankajgupta** [comment]
Yep. For comparison of scale up vs scale out, what is a distributed graph system du jour? Is it graphx?
